bundles/de.uka.ipd.sdq.simucomframework/src/de/uka/ipd/sdq/simucomframework/SimuComFactory.java

```diff
@@ -32,7 +34,8 @@
      *            Should desmoj trace our experiment
      * @return The created simulation model
      */
-    public static SimuComModel getSimuComModel(SimuComConfig config, SimuComStatus simuComStatus, boolean isRemote) {
+    public static SimuComModel getSimuComModel(SimuComConfig config, SimuComStatus simuComStatus, boolean isRemote
+            , IResourceTableManager resourceTableManager) {

```

Comment:
The line break looks missplaced. I'd expect the comma to be on the previous line.

Reply:
worked in review finding; reformatted file using Palladio code formatter